### PR TITLE
Task #169598 : Added post request and event service

### PR DESCRIPTION
**Pull request by Pratikshakhandagale**

### CODE REVIEW

File: projects/event-library/src/lib/events/services/event/event.service.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  items: any;
+  constructor(
+    private userConfigService: UserConfigService,
+    private dataService: DataService) {
+  }
+
+  /**
+   * To user enrolled event list
+   */
+  getEnrollEvents(courseId, userId) {
```
Comment (ahghatol) [inline projects/event-library/src/lib/events/services/event/event.service.ts:20]: @Pratikshakhandagale remove unused parameters

File: projects/event-library/src/lib/events/services/event/event.service.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      url: this.userConfigService.getConfigUrl().enrollListApi
+    };
+
+    return this.dataService.get(req);
+  }
+
+  /**
+   * For Enroll/Unenroll to the event
+   */
+  enrollToEventPost(action, cId, uId) {
```
Comment (ahghatol) [inline projects/event-library/src/lib/events/services/event/event.service.ts:32]: @Pratikshakhandagale Remove unused parameters

File: projects/event-library/src/lib/events/services/event/event.service.ts (hunk 1 above)
Comment (Pratikshakhandagale) [reply projects/event-library/src/lib/events/services/event/event.service.ts]: done

File: projects/event-library/src/lib/events/services/event/event.service.ts (hunk 2 above)
Comment (Pratikshakhandagale) [reply projects/event-library/src/lib/events/services/event/event.service.ts]: done

**Pull request merged**